clearblade.md

```diff
@@ -1323,6 +1323,59 @@ This function does not return query results.
 	db.exec("delete from traffic where entrance='Store_Entrance';");
 ~~~~
 
+## ClearBlade.Database.performOperation(callback, argument)
```

Comment:
Add jsdoc in this format:

https://github.com/ClearBlade/native-libraries/blob/master/clearblade.md#collectionfetchquery-callback